### PR TITLE
Closes #2353: Decreased padding to fix cut-off text

### LINKED ISSUE #2353: Settings elements names are cut in half
## Steps to reproduce
1. Launch FF for fire tv.
2. Scroll down until reaching the `Settings` section.
### Expected behavior
The `Settings` elements are displayed correctly(send usage data, clear all cookies, about, privacy notice)
### Actual behavior
The `Settings` elements names are cut in half.
### Device information
* Fire TV device: **`Fire Stick 2nd gen`**
* Latest affected Firefox version: **`v3.10-LAT3`**
* Earliest affected Firefox version: **`v3.10-LAT3`**

### PR DESCRIPTION
**Pull request by wmichelle024**
### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not

- UI fix - manually checked

- [x] This PR includes a **CHANGELOG entry** or does not need one
- [ ] I have considered adding **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

File: app/src/main/res/layout/settings_tile.xml [hunk 1]
```diff
@@ -35,7 +35,10 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:gravity="center"
-        android:padding="12dp"
```
Comment (Baron-Severin) [inline app/src/main/res/layout/settings_tile.xml]: Each commit should follow our commit format, would you mind amending the second?

Early commits will typically be prefaced with `Issue #<issue>:`, then `[Closes|Fixes] $<issue>` when you think the PR is complete.  It's also fine if the last few commits use `Closes`, due to review comments and other late changes.

Comment (Baron-Severin) [reply app/src/main/res/layout/settings_tile.xml]: It would also be fine to squash these into one commit.

Comment (wmichelle024) [reply app/src/main/res/layout/settings_tile.xml]: Hmm I'm not sure how to change a commit message that's already been pushed. Is this okay, or did you have something else in mind?

Comment (Baron-Severin) [reply app/src/main/res/layout/settings_tile.xml]: Are you still in the office?  If not we can go over it in person on Monday.  It's a simple process but it comes with some warnings.

**Pull request merged**